linera-views/src/dynamo_db.rs

```diff
@@ -131,8 +131,8 @@ impl LocalStackTestContext {
 /// The attribute name of the partition key.
 const PARTITION_ATTRIBUTE: &str = "item_partition";
 
-/// A dummy value to use as the partition key.
-const DUMMY_PARTITION_KEY: &[u8] = &[0];
+/// A root-key being used for testing existence of tables
+const DB_ROOT_KEY: &[u8] = &[];
```

Comment:
Would `EMPTY_ROOT_KEY` be a better name?

Reply:
Ok.